client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -568,6 +584,14 @@ export class WorkspaceView extends React.Component<any, any> {
       !isOracleWorkspace &&
       isParentOracleWorkspace;
 
+    const rootWorkspaceScratchPad = workspace.rootWorkspace.blocks.find(
+      b => b.type === "SCRATCHPAD",
+    );
+    const linkNodes = rootWorkspaceScratchPad.value[0].nodes.find(
+      n => n.type === "link",
+    );
+    const rootWorkspaceScratchPadLink = linkNodes ? linkNodes.data.href : "";
```

Comment:
I think we should remove this now that we're directly passing the block. Seems that this now sometimes throws an error:
![image](https://user-images.githubusercontent.com/1892071/62161680-9a661f80-b2cb-11e9-96a9-93e44698de0e.png)